types/office-js-preview/index.d.ts

```diff
@@ -37683,32 +37698,11 @@ declare namespace Excel {
          * @param color HTML color code representing the color of the background, in the form #RRGGBB (e.g., "FFA500") or as a named HTML color (e.g., "orange").
          */
         setSolidColor(color: string): void;
-        /**
```

Comment:
@wandyezj This block is removed from office-js-preview in the most recent CDN update, can you confirm it should be removed? 

`toJSON(): Excel.Interfaces.ChartFillData;` is not removed, so removing this block results in an npm lint error.